common/app/experiments/Experiments.scala

```diff
@@ -71,3 +72,12 @@ object ServerSideLiveblogInlineAds
       sellByDate = LocalDate.of(2023, 6, 1),
       participationGroup = Perc0A,
     )
+
+object TenImageSlideshows
+    extends Experiment(
+      name = "ten-image-slideshows",
+      description = "Test the impact of including 10 images in a slidehow rather than 5",
+      owners = Seq(Owner.withGithub("@guardian/editorial-experience")),
+      sellByDate = LocalDate.of(2023, 5, 15),
+      participationGroup = Perc50,
```

Comment:
Does this currently overlap with the DCR fronts test? I imagine they should be able to run along side one-another since DCR does not support slideshows yet, but just wanted to highlight this just in case